pkg/managerdriver/driver_test.go

```diff
@@ -62,7 +61,7 @@ var _ = Describe("Driver", func() {
 
 	Describe("Seed", func() {
 		It("Should not error", func() {
-			err := driver.Seed(context.Background(), fake.NewClientBuilder().WithScheme(scheme).Build())
+			err := driver.Seed(GinkgoT().Context(), fake.NewClientBuilder().WithScheme(scheme).Build())
```

Comment:
`GinkgoT().Context()` I used `ctx SpecContext`. I'm not sure the difference but 👍